lib/mtl/rails/view_helpers.rb

```diff
@@ -226,17 +226,21 @@ def mtl_button(label, url, options = {})
       # @param menu [String, Boolean] (HTML ID) references the id of the aside menu / default
       #   nav menu to show / hide on mobile devices. The default used is `nav-menu`.
       #   When set to `false`, this button is skipped.
+      # @param class [String, Array] (HTML CLASS) additional, custom css class on header
       # @yield Additional content to be rendered as part of the header
       # @return [String] HTML safe string
-      def mtl_header(title = translate('.title', default: 'Menu'),
-                     back: false, menu: 'nav-menu', &block)
+      def mtl_header(title = translate('.title', default: 'Menu'), **options, &block)
```

Comment:
the signature `mtl_header(title = ..., options = {}, &block)` won't pass the test because method calls like `mtl_header(menu: false)` will assign the hash to the `title` arguments, i.e `title = { menu => false }, options = {}`